plugin-utilities-core/src/main/kotlin/org/jetbrains/research/pluginUtilities/BuildSystems.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.research.pluginUtilities
+
+import org.jetbrains.research.pluginUtilities.util.subdirectories
+import java.io.File
+
+enum class BuildSystem(val buildFile: String) {
+    Maven("pom.xml"),
+    Gradle("build.gradle"),
+}
+
+val javaBuildSystems = listOf(BuildSystem.Maven, BuildSystem.Gradle)
```

Comment:
Please, don't use global variables. You can use arguments with default value in the functions